supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

```diff
@@ -4,6 +4,18 @@
 
 For more detailed guidance on how to document UI elements, see link:https://www.patternfly.org/v4/ux-writing/about[Patternfly].
 
+[[microcopy]]
+== Microcopy
+
+The words in a user interface, commonly referred to as "UX copy" or "microcopy", are just as important as the components or layouts. UX copy is another element of design, and it can drive better UX decisions and guide users to succeed. Red{nbsp}Hat cloud services are based on Patternfly, an open source design system created to enable consistency and usability across a wide range of applications and use cases.
```

Comment:
```suggestion
The words in a user interface, commonly referred to as "UX copy" or "microcopy", are just as important as the components or layouts. Microcopy is another element of design, and it can drive better UX decisions and guide users to succeed. Red{nbsp}Hat cloud services are based on PatternFly, an open source design system created to enable consistency and usability across a wide range of applications and use cases.
```
- I'd suggest sticking withe `Microcopy` because that's the heading name
- Change to `PatternFly`: https://pf4.patternfly.org/